docs/variants/asus_kgpe_d16/building-manual.md

```diff
@@ -0,0 +1,38 @@
+# Dasharo for Asus KGPE-D16 - building manual
+
+## Building coreboot
+
+To build coreboot image, follow the steps below:
+
+1. Clone the coreboot repository:
+
+    ```bash
+    $ git clone git@github.com:dasharo/coreboot.git -b asus_kgpe_d16/develop
+    ```
+
+2. Get the submodules:
+
+    ```bash
+    $ cd coreboot
+    $ git submodule update --init --recursive --checkout
```

Comment:
@miczyg1 It fails if we follow your instructions. The submodules have relative links. We do not add review.coreboot.org git. Submodules do not exist in Dasharo org.